### ISSUE TITLE
Accessibility - 1010EZ: Accessibility description on Continue and Back buttons is misleading

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#56858]
**Issue description**

![Birth information page on 1010EZ with red arrows pointing to h2 element on page and aria-describedby for Continue and Back buttons.](https://images.zenhubusercontent.com/628e7b9093a9366153cb221c/dbcc8387-fcbf-478e-84b7-7c80b6301855)

The Continue and Back buttons have an interesting pattern on them. They each have an aria-describedby linking to the h2 on the page, which is the step information, like “Step 1 of 6: Veteran information”, That means if a person using a screen reader hits one of those buttons, they’ll hear something like “Continue [pause] Step 1 of 6: Veteran information” But it’s misleading because it is announcing the current step, not the one you would go forward or back to by hitting the buttons.

This isn't a WCAG violation, more of an enhancement. In general, it's good to have descriptive buttons, but this feels like it could create confusion for people. We could maybe keep this pattern if there was a way to tell what the next or previous step was, and they were actually more descriptive. The other challenge here is that the step information doesn't change all that often. [See this related ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1538).

[See this discussion in Slack about the issue](https://dsva.slack.com/archives/C023ZKDJJK1/p1681409771825409).

**Remediation guidance**

Remove the `aria-describedby="nav-form-header"` on the Continue and Back buttons.

**Does this impact visual design?**

No.

**Relevant standards**

Enhancement.

**Comment by hdjustice** [comment]
This component is from Forms library and if it gets changed, it would change for all forms using it.  We cannot change anything until we confirm all users of this component approves it, as well as the Design System team.
Minimal effort to make the change.

**Comment by davidakennedy** [comment]
Just noting that I've validated this and it works as expected.
